File: Data Structures & Algorithms/remove-element/submission-1.js
class Solution {
    /**
     * @param {number[]} nums
     * @param {number} val
     * @return {number}
     */
    removeElement(nums, val) {
        let k = 0
        for(let j = 0; j < nums.length; j++){
            if(nums[j] !== val){
                nums[k] = nums[j]
                k++
            }
            
        }
        return k

    }
}

File: Data Structures & Algorithms/squares-of-a-sorted-array/submission-1.py
class Solution:
    def sortedSquares(self, nums: List[int]) -> List[int]:
        squares = [0] * len(nums)

        left = 0

        right = len(nums) - 1


        for i in range(len(nums) - 1, -1, -1):
            
            if abs(nums[left]) > abs(nums[right]):

                squares[i] = nums[left] * nums[left]

                left +=1

            else:
                squares[i] = nums[right] * nums[right]

                right -=1

        return squares

        
        